frontend/components/CategorySelectionOnFnd.tsx

```
@@ -0,0 +1,150 @@
+  id: number;
+  name: string;
+  skills: string[];
+}
+
+const CategorySelectionOnFnd = () => {
+  const [selectedCategory, setSelectedCategory] = useState<Category | null>(null);
+  const [selectedSkills, setSelectedSkills] = useState<string[]>([]);
+  const [error, setError] = useState("");
+
+  const handleCategoryChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const categoryId = parseInt(event.target.value);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)